R/summarize_dt.R

```diff
@@ -11,8 +11,10 @@
 #' @param summarize_cols \[`character()`\]\cr
 #'   The `id_cols` that should be collapsed and to calculate summary statistics
 #'   over.
-#' @param value_col \[`character(1)`\]\cr
-#'   Value column that summary statistics should be calculated for.
+#' @param value_cols \[`character()`\]\cr
+#'   Value columns that summary statistics should be calculated for. When more
```

Comment:
Or should this be formatted long? The reason I am working on this was for updating `gen_summary_lt` https://github.com/ihmeuw-demographics/demCore/issues/67

Reply:
Nevermind I guess this still allows users to use the function with long formatted data. And we just need to separately decide if `gen_summary_lt` should still return data long with the extra `life_table_parameter` column

Reply:
Do you need to edit the `@return` field too? To specify output will be wide on value column and summary function?